src/agent/adu_core_interface/tests/adu_core_interface_ut.cpp

```diff
@@ -70,21 +71,21 @@ class ADUC_UT_ReportPropertyAsyncValues
     void
     set(ADUC_ClientHandle deviceHandleIn,
         const unsigned char* reportedStateIn,
-        size_t reportedStateLenIn,
+        size_t _reportedStateLenIn,
         IOTHUB_CLIENT_REPORTED_STATE_CALLBACK reportedStateCallbackIn,
         void* userContextCallbackIn)
     {
         deviceHandle = deviceHandleIn;
 
-        memcpy(reportedState, reportedStateIn, reportedStateLenIn);
+        reportedState = reinterpret_cast<const char*>(reportedStateIn);
```

Comment:
>reinterpret_cast [](http://example.com/codeflow?start=24&length=16)

did you run this through our scripts?  Seems like lint would flag this.

See [prior to submitting a PR]([https://microsoft.sharepoint.com/teams/DSPlat/_layouts/OneNote.aspx?id=%2Fteams%2FDSPlat%2FShared%20Documents%2FEngineering%20Team%2FUpdate%20Client%2FADUC%2FAzure%20Device%20Update%20Client%20%28ADUC%29&wd=target%28For%20Developers.one%7C3141E8F5-E198-4573-9A5D-40A0C950A53F%2FPull%20Request%5C%2FCode%20Review%20Guidelines%7C9672469B-9896-4BD1-9351-C6055B6B001C%2F%29](https://microsoft.sharepoint.com/teams/DSPlat/_layouts/OneNote.aspx?id=%2Fteams%2FDSPlat%2FShared%20Documents%2FEngineering%20Team%2FUpdate%20Client%2FADUC%2FAzure%20Device%20Update%20Client%20%28ADUC%29&wd=target%28For%20Developers.one%7C3141E8F5-E198-4573-9A5D-40A0C950A53F%2FPull%20Request%5C%2FCode%20Review%20Guidelines%7C9672469B-9896-4BD1-9351-C6055B6B001C%2F%29)
[onenote:https://microsoft.sharepoint.com/teams/DSPlat/Shared%20Documents/Engineering%20Team/Update%20Client/ADUC/Azure%20Device%20Update%20Client%20(ADUC)/For%20Developers.one#Pull%20Request/Code%20Review%20Guidelines&section-id={3141E8F5-E198-4573-9A5D-40A0C950A53F}&page-id={9672469B-9896-4BD1-9351-C6055B6B001C}&end](onenote:https://microsoft.sharepoint.com/teams/DSPlat/Shared%20Documents/Engineering%20Team/Update%20Client/ADUC/Azure%20Device%20Update%20Client%20(ADUC)/For%20Developers.one#Pull%20Request/Code%20Review%20Guidelines&section-id={3141E8F5-E198-4573-9A5D-40A0C950A53F}&page-id={9672469B-9896-4BD1-9351-C6055B6B001C}&end))

Reply:
I neglected to re-run static-analysis after the new iteration commit.  Will do so going forward.